mmv1/third_party/terraform/services/workbench/resource_workbench_instance_test.go.tmpl

```diff
@@ -350,7 +350,7 @@ func TestAccWorkbenchInstance_updateMetadataKey(t *testing.T) {
 				ResourceName:            "google_workbench_instance.instance",
 				ImportState:             true,
 				ImportStateVerify:       true,
-				ImportStateVerifyIgnore: []string{"name", "instance_owners", "location", "instance_id", "request_id", "labels", "terraform_labels"},
+				ImportStateVerifyIgnore: []string{"name", "instance_owners", "location", "instance_id", "request_id", "labels", "terraform_labels", "update_time"},
```

Comment:
Per https://github.com/hashicorp/terraform-provider-google/issues/22220 the health_info and health_state fields are also sometimes impacted.

```suggestion
				ImportStateVerifyIgnore: []string{"name", "instance_owners", "location", "instance_id", "request_id", "labels", "terraform_labels", "update_time", "health_info", "health_state"},
```